package.json

```diff
@@ -19,23 +19,25 @@
   "scripts": {
     "format": "prettier --write ./bin ./lib ./src ./test",
     "lint": "eslint ./bin/**.cjs ./lib/**.cjs ./src/**.js ./test/**.cjs ",
-    "test": "pnpm test:unit && tape './test/*.cjs'",
+    "test": "npm run test:unit && tape './test/*.cjs'",
     "test:unit": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
-    "demo": "cd ./test/demo && pnpm start",
+    "demo": "cd ./test/demo && npm start",
     "build": "esbuild src/index.js --bundle --minify --platform=node --outfile=./dist/index.cjs"
   },
+  "ManifestComments": [
+    "Pinned update-notifier to 5.1.0, v6+ requires ESM"
+  ],
```

Comment:
Unfortunately JSON files do not support comments. So I've added this section here, so documentation around the `package.json` file can still occur.

If you think `package.json` files should be more permissive to allow for comments, you can +1 this post:

* https://github.com/npm/rfcs/issues/291